partner_daytopay/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Open Source Integrators <https://www.opensourceintegrators.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Partner Days to Pay',
+    'summary': 'Add receivables and payables statistics to partners',
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Open Source Integrators, Odoo Community Association (OCA)',
+    'category': 'Accounting & Finance',
+    'website': 'http://www.opensourceintegrators.com',
```

Comment:
As per new __manifest__ , can you change the information, and website?
See https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py